backend/curation/api/v1/curation/collection_versions/collection_version_id/actions.py

```diff
@@ -0,0 +1,23 @@
+from flask import jsonify
+
+from backend.common.utils.http_exceptions import ForbiddenHTTPException, NotFoundHTTPException
+from backend.curation.api.v1.curation.collections.common import (
+    reshape_for_curation_api,
+    validate_uuid_else_forbidden,
+)
+from backend.layers.common.entities import CollectionVersionId
+from backend.portal.api.providers import get_business_logic
+
+
+def get(collection_version_id: str):
+    """
+    Fetch the full metadata of a published CollectionVersion, including its datasets
+    """
+    validate_uuid_else_forbidden(collection_version_id)
+    version = get_business_logic().get_collection_version(CollectionVersionId(collection_version_id))
+    if version is None or version.canonical_collection.tombstoned is True:
+        raise NotFoundHTTPException()
```

Comment:
tombstones should produce a 403

Reply:
have not done an exhaustive search for consistency, might be worth doing, but see [this](https://github.com/chanzuckerberg/single-cell-data-portal/blob/eefb7d71bb50fe81f434d652c51a09ec4bc344dd/backend/curation/api/v1/curation/collections/common.py#L302-L303) as example